server/src/test/scala/RoutesTests.scala

```diff
@@ -0,0 +1,18 @@
+package server
+
+import org.scalatest.FunSuite
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+
+class RoutesTests extends FunSuite with ScalatestRouteTest {
+  test("/") {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal "/" appears 2 times in the file.](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594775830/issues/source?bid=3301982&fileBranchId=3306761#l7)